### ISSUE TITLE
@ DankNetCode (bitcointalk)

### ISSUE THREAD

**Issue by nemosminer** [nemosminer/NemosMiner-v2.5.1#329]
Tried to PM this to you, but my account is too newb to get it through.  I guess this isn't entirely a hi-jacking since it's somewhat NM related.

Put this into the arguments line of each ps1 miner file you use in the "miners" folder in nemos:

"-b 0.0.0.0:4068" (without the quotes)

I'm not sure why the --api remote argument doesn't work.  I think it might work if you were running AM locally on your mining rig since it defaults to a loopback IP (127.0.0.1).  I'm sure there's a way to get it to work using additional arguemtns, but the above argument worked for me right out of the gate so I just stuck with it.  

Depending on what firewall and OS you're running, the first time each miner runs, you'll likely be prompted to acknowledge the firewall exceptions.  Certainly happens on my W10, Windows defender rig.

Cheers.

J.

**Comment by mynameisrich** [comment]
agreed about the quickedit property.. i have definitely fallen for that a few times

**Comment by valth0nis** [comment]
Seconded about the quickedit property.
